src/main/java/gyro/aws/ec2/PeeringConnectionResource.java

```diff
@@ -255,25 +259,21 @@ protected void doCreate(GyroUI ui, State state) {
         VpcPeeringConnection vpcPeeringConnection = response.vpcPeeringConnection();
         setId(vpcPeeringConnection.vpcPeeringConnectionId());
 
-        client.acceptVpcPeeringConnection(
-            r -> r.vpcPeeringConnectionId(getId())
-        );
-
+        waitForStatus(client, "pending-acceptance");
```

Comment:
Add a save to persist state, in case of exceptions in the modify or the wait
```suggestion
        state.save();
        waitForStatus(client, "pending-acceptance");
```